app/models/classroom.rb

```diff
@@ -66,6 +68,18 @@ def class_time
     start_time.strftime('%H:%M') + ' - ' + end_time.strftime('%H:%M')
   end
 
+  def minified_url_for_teacher(teacher)
+    one_time_password = rand(36**10).to_s(36)
+    teacher.update(password: one_time_password)
+    url_suffix = "/classrooms/#{id}?email=#{teacher.email}&password=#{one_time_password}"
```

Comment:
From my understanding, this will keep the email/password info in the URL as a query string? Do we want it to stay there?

Reply:
for now it shouldn't be a big issue as the password gets reset every time a new session is scheduled, but we should definitely think about a way how to improve this

Reply:
could have it go to one action for authenticating and then redirect after to another 🤔